### PR TITLE
Add minitest-flash to continuous testing

### PR DESCRIPTION
**Pull request by svoop**
Thanks for contributing to the Ruby Toolbox catalog! <3

**Before submitting your pull request:**

- [x] If you're referencing a gem by its GitHub repository (e.g. `rails/rails`), please verify
      that it is _not_ packaged as a Ruby gem. (If it _is_ packaged as a gem, please reference it
      by its gem name instead, e.g. `rails`.)
- [x] Please make sure the projects are listed in case-insensitive alphabetical order
- [x] Make sure the CI build passes, we validate your proposed changes in the test suite :)

### CODE REVIEW

File: catalog/Testing/continuous_testing.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ projects:
   - guard-rubocop
   - infinity_test
   - kicker
+  - minitest-flash
+  - minitest-substitute
```
Comment (colszowka) [inline catalog/Testing/continuous_testing.yml:13]: @svoop We already have minitest-substitute in the mocking category, and it seems more fitting there, I think we should just put minitest-flash into this category, WDYT?

Comment (svoop) [reply catalog/Testing/continuous_testing.yml:13]: Oh, my bad @colszowka, I was in such a flow. :smile: I'll force push it out of this PR.

Comment (svoop) [reply catalog/Testing/continuous_testing.yml]: There you go, the dupe is removed.

**Comment by colszowka** [comment]
Thanks @svoop !

**Pull request merged**